src/api/java/com/minecolonies/api/colony/IColony.java

```diff
@@ -308,6 +310,9 @@
 
     void increaseBoughtCitizenCost();
 
+    @NotNull
+    Set<EntityPlayer> getImportantMessageEntityPlayers();
```

Comment:
Please add documentation to new methods that are part of the API.

Reply:
accidently added the doc to it on a different branch, so I'll leave it empty on this to not have a merge conflict